packages/apptive_grid_form/lib/src/widgets/form_widget/form_content_widgets.dart

```diff
@@ -0,0 +1,395 @@
+import 'package:apptive_grid_form/apptive_grid_form.dart';
+import 'package:apptive_grid_form/src/translation/apptive_grid_localization.dart';
+import 'package:apptive_grid_form/src/util/submit_progress.dart';
+import 'package:apptive_grid_form/src/widgets/apptive_grid_form_widgets.dart';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'package:lottie/lottie.dart';
+
+/// Shows a spinner while loading the form data
+class LoadingFormWidget extends StatelessWidget {
+  /// Creates a new loading widget
+  const LoadingFormWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Center(
+      child: CircularProgressIndicator.adaptive(),
+    );
+  }
+}
+
+/// Displays the current form data with all customizations
+class FormDataWidget extends StatelessWidget {
```

Comment:
This should still be in the other file

Reply:
Why? This is part of the widgets that have been extracted from a builder method

Reply:
Yes but they have never been public available (Except from the FormDataWidget) and I don't see a reason to start doing so right now.

Reply:
The FormDataWidget was never public since it didn't exist before, I think you are thinking of ApptiveGridFormData

Reply:
Ah yeah 😂 got confused with all these names then